src/WorkOS.net/Services/UserManagement/UserManagementService.cs

```diff
@@ -64,6 +64,25 @@ public UserManagementService(WorkOSClient client)
             return await this.Client.MakeAPIRequest<User>(request, cancellationToken);
         }
 
+        /// <summary>
+        /// List Users.
+        /// </summary>
+        /// <param name="options"> Parameters to filter list of users.</param>
+        /// <param name="cancellationToken">An optional token to cancel the request.</param>
+        /// <returns> A list of Users.</returns>
+        public async Task<WorkOSList<User>> ListUsers(
+            ListUsersOptions options,
+            CancellationToken cancellationToken = default)
+        {
+            var request = new WorkOSRequest
+            {
+                Options = options,
```

Comment:
One thing we should do is set the `order` query parameter to `desc` by default so that we can take advantage of the fixed pagination order.

Reply:
Since this version is a breaking change, do we want to set this as default at the ListOptions interface and force everyone to be on the fixed pagination order by default? 
![CleanShot 2023-08-22 at 16 21 59@2x](https://github.com/workos/workos-dotnet/assets/5375567/aa2ebeb8-dc85-456b-a0dc-f2d2652518a3)

Reply:
That sounds like a good idea!

I think we can make `PaginationOrder` non-nullable too, so that we always provide a value:

```cs
public PaginationOrder Order { get; set; } = PaginationOrder.Desc;
```